### ISSUE TITLE
Type error Elevenlabs

### ISSUE THREAD

**Issue by balogistvan** [livekit/agents#743]
Hi there. I have a problem, when I add TTS with elevenlabs plugin, I get this error:
Plugin.register_plugin(ElevenLabsPlugin())
TypeError: Can't instantiate abstract class ElevenLabsPlugin with abstract method download_files

livekit version: 0.15.2 but the error still has in version 0.12.0

**Comment by davidzhao** [comment]
I don't believe that line is necessary. Also, what is version `0.15.2`?  The latest release of agent framework is 0.8.12
